src/pages/Pool.vue

```diff
@@ -159,13 +155,32 @@ export default defineComponent({
       refetchQueriesOnBlockNumber: 0
     });
 
-    const feesManagedByGauntlet = POOLS.DynamicFees.Gauntlet.includes(data.id);
-
     // COMPUTED
     const appLoading = computed(() => store.state.app.loading);
 
     const pool = computed(() => poolQuery.data.value);
 
+    const communityManagedFees = computed(
+      () => pool.value?.onchain.owner == POOLS.DelegateOwner
+    );
+    const feesManagedByGauntlet = computed(
+      () =>
+        communityManagedFees.value &&
+        POOLS.DynamicFees.Gauntlet.includes(data.id)
+    );
+    const feesFixed = computed(
+      () => pool.value?.onchain.owner == POOLS.ZeroAddress
+    );
+    const swapFeeToolTip = computed(() =>
+      feesManagedByGauntlet.value
+        ? t('feesManagedByGauntlet')
+        : feesFixed.value
+        ? t('fixedFeesTooltip')
+        : communityManagedFees.value
+        ? t('delegateFeesTooltip')
+        : t('ownerFeesTooltip')
```

Comment:
Using the ternary operator is quite unreadable here. 
```suggestion
      if (feesManagedByGauntlet.value) {
        t('feesManagedByGauntlet')
      } else if (communityManagedFees.value)
        t('delegateFeesTooltip')
      } else if (feesFixed.value)
        t('fixedFeesTooltip')
      } else {
        t('ownerFeesTooltip')
      }
```